pygmt/tests/test_sphinterpolate.py

```diff
@@ -0,0 +1,34 @@
+"""
+Tests for sphinterpolate.
+"""
+import os
+
+import numpy.testing as npt
+from pygmt import sphinterpolate
+from pygmt.helpers import GMTTempFile
+
+
+def test_sphinterpolate_outgrid():
+    """
+    Test sphinterpolate with a set outgrid.
+    """
+    with GMTTempFile(suffix=".nc") as tmpfile:
+        result = sphinterpolate(
+            table="@mars370d.txt", outgrid=tmpfile.name, spacing=1, region="g"
+        )
+        assert result is None  # return value is None
+        assert os.path.exists(path=tmpfile.name)  # check that outgrid exists
+
+
+def test_sphinterpolate_no_outgrid():
+    """
+    Test sphinterpolate with no set outgrid.
+    """
+    temp_grid = sphinterpolate(table="@mars370d.txt", spacing=1, region="g")
```

Comment:
Will leave it up to you on how to include `load_mars_dataset` from #1420 into this test.
```suggestion
    temp_grid = sphinterpolate(data="@mars370d.txt", spacing=1, region="g")
```

Reply:
changed in a7b12e3